KTLOSNaxxLoot.json

```diff
@@ -53,7 +53,7 @@
 			"wowID":""
 		},
 		"Desecrated Bracers":{
-			"prio":["4h Tanks","LC","EP/GP"],
+			"prio":["4h Tanks","LC","Bendriller(flex)","Johnscar(crush)","EP/GP"],
```

Comment:
why are you adding yourself first after not showing up for 2 weeks?

Reply:
I thought I only missed 1 week? (1/3/2020) Johnscar has prio for 4 pieces and I also have prio for a specific 4 piece that I need to run to have the best 4/4 split with darkmantle. Killstep wants to wait on tier so other people can have it first.

Reply:
Oh you're right, put yourself as (sunnyD) instead of flex then.
I'm ok with this, but I don't think you should get T3 shoulders / legs over someone like Sniped who is on 3 pieces, and just needs that to complete 4.

Reply:
good point, it seems like we have more rogues than can fit in raid 1 right now so I wasn't planning to swap raids any time soon.

Reply:
For the record I'm not entirely happy if Xlord / Devilspebble get t3 before me just because they are bitchy about what raid they go to, but it's not a big deal.